latest/docs-ref-conceptual/get-started-with-azure-cli.md

```diff
@@ -1,442 +1,88 @@
-
-Like with the create command, you can list resources using the Azure CLI 2.0 using a common `az <resource type name> list` naming pattern that is consistent across all resource types.  There are various output formats and query options available to filter and sort the list of resources in the way you prefer to see them.
-
-For example, `az vm list` shows the list of all VMs you have.   
-
-```azurecli-interactive
-az vm list 
+```azurecli
+az login
 ```
-The values returned are by default in JSON (only showing partial output for sake of brevity).
 
-```json
-[
-  {
-    "availabilitySet": null,
-    "diagnosticsProfile": null,
-    "hardwareProfile": {
-      "vmSize": "Standard_DS1_v2"
-    },
-    "id": "/subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/DEMORG1/providers/Microsoft.Compute/virtualMachines/DemoVM010",
-    "instanceView": null,
-    "licenseType": null,
-    "location": "westus2",
-    "name": "MyLinuxVM",
-    "networkProfile": {
-      "networkInterfaces": [
-        {
-          "id": "/subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/demorg1/providers/Microsoft.Network/networkInterfaces/DemoVM010VMNic",
-          "primary": null,
-          "resourceGroup": "MyResourceGroup"
-        }
-      ]
-    },
-          ...
-          ...
-          ...   
-]
-```
+This provides a link to the Azure CLI authentication portal, and a one-time code to authorize with. Authentication expires
+if it is not used for a period of 14 days. There are other advanced authentication methods that can be used once
+you have configured the appropriate Azure resources:
 
-You can optionally modify the output format using the `--output` option.  Run the `az vm list` command to see both the Linux and Windows Server VMs created earlier, along with the most common properties of a VM, using the easy to read *table* format option:
+* For logging in with Managed Service Identities (MSI) see [Configure MSI for an Azure VM](/azure/active-directory/msi-qs-configure-cli-windows-vm?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json) and [Sign in on an Azure VM using MSI](/azure/active-directory/msi-how-to-get-access-token-using-msi?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json)
+* For logging in with an Active Directory service principal, see [Create a service principal](create-an-azure-service-principal-azure-cli.md)
 
-```azurecli-interactive
-az vm list --output table
-```
+## Common operations with Azure
 
-```Output
-Name       ResourceGroup    Location
----------  ---------------  ----------
-MyLinuxVM  MyResourceGroup  westus2
-MyWinVM    MyResourceGroup  westus2
-```
+To help with getting started quickly, here is a list of some common operations performed with the CLI. Any of these can have their help information printed with the `---help` option. There is also the online [Azure CLI reference documentation](/cli/azure/).
```

Comment:
Consider a change like this which simplifies the sentences to make your message more clear:

> To get started quickly, the following table lists common operations performed with the CLI. Documentation for the commands are available online via the [Azure CLI reference documentation](/cli/azure/) or through the command prompt by adding the `---help` option after any of the commands.